bin/docker-entrypoint

```diff
@@ -6,14 +6,17 @@
 # all groupIDs from the host users are added to this user.
 #
 
+# Exit if a command exits with a non-zero status
+set -e
+
 # Add and run as local user if LOCAL_USER_ID is passed in at runtime
 if [ -z "$LOCAL_USER_ID" ]; then
   echo "LOCAL_USER_ID must be set in the environment when running this container, exiting."
   exit 1
 fi
 
 # create user with $LOCAL_USER_ID if it doesn't exists
-if [[ ! $(id -u "$LOCAL_USER_ID" > /dev/null 2>&1) ]]; then
+if ! id -u "$LOCAL_USER_ID" > /dev/null 2>&1; then
```

Comment:
Why removing the brackets?

Reply:
The brackets are unnecessary when checking for exit code (and the previous check was wrong....:)

Reply:
This made image not run if id was there already - as was case when running as sudo (id=0 already existed)